src/ui/Pricing/Comparison/Table.svelte

```diff
@@ -41,6 +41,8 @@
 
   .link {
     height: 26px;
+    --accent: var(--green);
+    --accent-hover: var(--green-hover);
```

Comment:
`[D-1-1]` Accents are defined in the [project's root css rule](https://github.com/santiment/app/blob/11e943992bf5a079e12f66b85e281b409f2bf597/src/index.scss#L45-L47), which uses `san-webkit` library
```suggestion
```

Reply:
It is overwritten on the upper component due to current Pricing page elements design. The same is for MoreFeaturesButton
[Here](https://github.com/santiment/san-webkit/blob/cba8c88455f7610d57fbcbee917f1ac724e57182/src/ui/Pricing/Comparison/index.svelte#L51)

Reply:
I see. Thanks for pointing it out